indico/modules/events/registration/templates/management/regform_stats.html

```diff
@@ -0,0 +1,213 @@
+                                {%- trans places=total-taken %}{{ places }} places available{% endtrans -%}
+                            {% endif -%}
+                        </span>
+                    </span>
+                </span>
+            </div>
+        {% endif %}
+        {% if stats.num_countries %}
+            <div class="stats-item">
+                <span>{% trans %}Registrants per country{% endtrans %}</span><span></span>
+            </div>
+            <div id="countriesPlot" style="width:500px; height:{{height}}px;"></div>
+            <script>
+                $(document).ready(function generateBasicStats() {
+                    var countries = {{ stats.countries|tojson }};
+                    var jqPlotOptions = {
+                        axes: {
+                            xaxis: {
+                                min: 0,
+                                padMax: 0,
+                            },
+                            yaxis: {
+                                pad: 8,
+                                renderer: $.jqplot.CategoryAxisRenderer,
+                                tickOptions: {
+                                    showGridline: false,
+                                    showLabel: true,
+                                    tickSpacing: 24
+                                }
+                            },
+                        },
+                        height: {{ height }},
+                        highlighter: {
+                            showMarker: false,
+                            showTooltip: false
+                        },
+                        seriesDefaults: {
+                            renderer: $.jqplot.BarRenderer,
+                            rendererOptions: {
+                                barDirection: 'horizontal',
+                                barWidth: 18,
+                                shadow: true,
+                                shadowDepth: 3,
+                                shadowOffset: 1
+                            },
+                            pointLabels: { show: true }
+                        },
+                    }
+                    var countriesPlot = $.jqplot('countriesPlot', [countries], processJqPlotOptions(jqPlotOptions));
+                });
+            </script>
+        {% endif %}
+    {% endcall %}
+{% endmacro %}
+
+{% macro render_table(table) -%}
+    <table class="registrant-stats">
+        {%- if table.head %}
+            <thead>
+                <tr>
+                {%- for head in table.head %}
+                    {% if head.type == 'str' -%}
+                        <th {%- if head.colspan > 1 %} colspan="{{ head.colspan }}"{% endif %}>{{ head.data }}</th>
+                    {%- endif -%}
+                {% endfor %}
+                </tr>
+            </thead>
+        {%- endif %}
+        {%- if table.rows %}
+            <tbody>
+            {%- for row_type, row in table.rows %}
```

Comment:
should have one more level of indentation (probably it was already wrong in jacques' code though)